File: app/src/main/java/com/example/eduempoweryd/login/registerpage/RegisterPageVM.java
package com.example.eduempoweryd.login.registerpage;

import android.os.Bundle;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class RegisterPageVM extends ViewModel {
    public MutableLiveData<RegisterPageModel> RegisterPageModel = new MutableLiveData<>(new RegisterPageModel());
    private MutableLiveData<Bundle> navArguments = new MutableLiveData<>();

    public MutableLiveData<RegisterPageModel> getPageTwoModel() {
        return RegisterPageModel;
    }

    public MutableLiveData<Bundle> getNavArguments() {
        return navArguments;
    }

    public void setNavArguments(Bundle navArguments) {
        this.navArguments.setValue(navArguments);
    }
}

File: jia_hui_s_application/app/src/main/java/com/jiahuisapplication/app/modules/registerpage/data/viewmodel/RegisterPageVM.kt
package com.jiahuisapplication.app.modules.registerpage.`data`.viewmodel

import android.os.Bundle
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.jiahuisapplication.app.modules.registerpage.`data`.model.RegisterPageModel
import org.koin.core.KoinComponent

class RegisterPageVM : ViewModel(), KoinComponent {
  val registerPageModel: MutableLiveData<RegisterPageModel> = MutableLiveData(RegisterPageModel())

  var navArguments: Bundle? = null
}
